src/test/java/com/google/sps/VerificationTest.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.when;
+import com.google.sps.data.*;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.servlets.*;
+import java.util.*;
+import java.lang.Long;
+
+//Tests user Verification Logic
+@RunWith(JUnit4.class)
+public final class VerificationTest {
+
+   @Test
+   public void userIsaMember(){
+        //Setup of Room
+        List<Member> members = new LinkedList<Member>(Arrays.asList(Member.createNewMember("RJ"), Member.createNewMember("RMJ"), Member.createNewMember("@google.com"), Member.createNewMember("rossjohnson@google.com")));
```

Comment:
Check out JUnit4 built in Setup function and you can move Room creation steps in the common setup function

Reply:
The rooms have different member lists, and after doing some reading I'm not sure if the common setup function would work.